clickhouse-http-client/src/main/java/com/clickhouse/client/http/HttpUrlConnectionImpl.java

```diff
@@ -101,11 +99,48 @@ private ClickHouseHttpResponse buildResponse(ClickHouseOutputStream output, Runn
                 displayName, queryId, summary, format, timeZone);
     }
 
+    private HttpURLConnection dispatchProxy(ClickHouseProxyType proxyType) throws IOException {
```

Comment:
Please add `url` argument to fix the test failure. The protected member `url` was reserved only for reusable connection.

Reply:
@zhicwu Will fix the integration tests? (they keep failing also when running them on main it fails )